### ISSUE TITLE
my build is suddently failing

### ISSUE THREAD

**Issue by GuyMograbi** [angular-ui/bootstrap#4566]
```
        TypeError: 'undefined' is not a function (evaluating 'dropdownCtrl.init()')
            at /home/home/dev_env/projects_GIT/app/bower_components/angular-bootstrap/ui-bootstrap-tpls.js:2976
```

I have several tests failing with the same stacktrace. 
I was able to reproduce this after doing bower cache clean and install
do you have any idea what could cause this?

**Comment by wesleycho** [comment]
Please post a minimal reproduction. In this case, even using the deprecated `dropdown` directive set works fine, as can be seen [here](http://plnkr.co/edit/Ojj0ExaDATnA7fR9TU5D?p=preview).

**Comment by GuyMograbi** [comment]
This is my code

```
    <div class="user-section-menu" dropdown is-open="menuIsOpen">
        <ul class="dropdown-menu">
            <li><a href="/#!/user/create/profile">{{'baseLayout.header.links.profile' | translate}}</a></li>

        </ul>
    </div>
```

if I remove the directive, the test passes. 
otherwise the above exception is thrown. 

downgrading fixes issue as well.

**Comment by GuyMograbi** [comment]
It fails on page load as well.. 

I am still working on minimal code to reproduce.. 

![image](https://cloud.githubusercontent.com/assets/287367/10411116/1713c974-6f64-11e5-9f21-af8477e39554.png)

**Comment by GuyMograbi** [comment]
reproduction was very easy... 

http://embed.plnkr.co/5XaONAFPW9KUayzjR9q6/preview

seems to me like you have a bug in 0.14.0.

**Comment by Foxandxss** [comment]
I'll check later :)

**Comment by johntmyers** [comment]
Same here. Doesn't look like a handful of directives were actually made backwards compatible with 0.13.x.

**Comment by wesleycho** [comment]
It should be noted that we are only actively testing with the latest release of angular 1.4 - we do support 1.3, but there is a very subtle issue in how Angular 1.3 is handling controller instantiation it looks like.

**Comment by wesleycho** [comment]
Opened an issue with Angular in https://github.com/angular/angular.js/issues/13069, I believe this should be fixed there.

**Comment by wesleycho** [comment]
This is now fixed in 0.14.1
